src/scenes/Prizes/index.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Backdrop from 'components/Backdrop';
+import { getPrizes } from 'api';
+
+import prizesFront from 'assets/common_backdrop/prizes-front.svg';
+
+import './style.scss';
+
+export default class Prizes extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      prizes: [],
+    };
+  }
+
+  componentDidMount() {
+    getPrizes().then(list => {
+      this.setState({ prizes: list.prizes });
+    });
+  }
+
+  render() {
+    const prizesList = this.state.prizes;
+
+    return (
+      <Backdrop
+        title="Prizes"
+        signs={[]}
```

Comment:
The signpost is awkwardly tall imo. See if you can shrink it based on how many signs it has

Reply:
I'll look into it, but I didn't bother changing it since the signpost was the same height regardless of there being 3 or 4 different things on the other pages

Reply:
should be fixed, affects all backdrop pages though